dragonchain/exceptions.py

```diff
@@ -167,5 +167,9 @@ class MatchmakingError(DragonchainException):
     """Exception raised by matchmaking client when a problem has occurred"""
 
 
+class RetryableError(DragonchainException):
```

Comment:
If this is specifically for matchmaking, it should probably be something like `MatchmakingRetryableError`

Also anywhere that used to do `except MatchmakingError` will now need to also catch this error too

Reply:
Good point, I'll rename it.  
Looks like the only other spots catching `MatchmakingError` exceptions are `execute()` in `level_1_actions.py` and `update_registration()` in `matchmaking.py` ...